src/extension.ts

```diff
@@ -262,6 +267,19 @@ function makeEndpointsProvider(): EndpointsProvider {
     return provider
 }
 
+function recordToolkitInitialization(activationStartedOn: number, logger?: Logger) {
+    try {
+        const activationFinishedOn = Date.now()
+        const duration = Math.abs(activationFinishedOn - activationStartedOn)
```

Comment:
If this is negative we have some large issues, can we not use abs? If we start getting negative numbers we will know there is something wrong, otherwise we might just think it is OK

Reply:
Negative values would indicate something happened on the host system outside the toolkit's control (maybe a time-sync occurred for example). In these scenarios, this value is likely no longer accurate. Should we send the pure value and filter out when consuming the data?